components/06-examples/beheer-basis.config.yml

```diff
@@ -0,0 +1,60 @@
+notes: |
+status: wip
+context:
+  menu: 
+    modifier: main
+    items: 
+      - label: Menu titel A
```

Comment:
Heb je bewust voor een eigen menu gekozen? Als je in `beheer-basis.hbs` de `{{> '@header' }}` vervang met `{{render '@header' }}` (en je tevreden bent met de inhoud) kan je de menu specificatie hier weg laten.

Reply:
Het menu is bewust aangepast. De standaard header bevat het menu van het Omgevingsloket. Dit leek mij verwarrender uitzien in het voorbeeld voor het beheerscherm, dan deze opzet. Tnx voor de tip met `{{render '@header' }} `trouwens.